web/src/lib/components/photos-page/memory-lane.svelte

```diff
@@ -45,7 +45,7 @@
     onscroll={onScroll}
   >
     {#if canScrollLeft || canScrollRight}
-      <div class="sticky start-0">
+      <div class="sticky start-0 z-1">
```

Comment:
We're getting into a bad habbit again where the solution is to just slap on z-indices 😅

Reply:
Nope, already try other solutions